Documentation/Benchmarks.md

```diff
@@ -43,27 +47,27 @@ AVERAGE                   | 5.6    | 2.3    | 142.6   | 6545.5s  | 607       | 2
 Dataset: https://github.com/Lab41/VOiCES-subset
 
 ```text
-swift run fluidaudio vad-benchmark --dataset voices-subset --all-files --threshold 0.5
+swift run fluidaudio vad-benchmark --dataset voices-subset --all-files --threshold 0.85
 ...
-⏱️ Timing Statistics:
-[23:26:10.167] [INFO] [VAD]    Total processing time: 2.76s
-[23:26:10.167] [INFO] [VAD]    Total audio duration: 350.46s
-[23:26:10.167] [INFO] [VAD]    RTFx: 126.9x faster than real-time
-[23:26:10.167] [INFO] [VAD]    Audio loading time: 0.01s (0.2%)
-[23:26:10.167] [INFO] [VAD]    VAD inference time: 2.75s (99.7%)
-[23:26:10.167] [INFO] [VAD]    Average per file: 0.115s
-[23:26:10.167] [INFO] [VAD]    Min per file: 0.022s
-[23:26:10.167] [INFO] [VAD]    Max per file: 0.135s
-[23:26:10.167] [INFO] [VAD]
-📊 VAD Benchmark Results:
-[23:26:10.167] [INFO] [VAD]    Precision: 100.0%
-[23:26:10.167] [INFO] [VAD]    Accuracy: 100.0%
-[23:26:10.167] [INFO] [VAD]    Recall: 100.0%
-[23:26:10.167] [INFO] [VAD]    F1-Score: 100.0%
-[23:26:10.167] [INFO] [VAD]    Total Time: 2.76s
-[23:26:10.167] [INFO] [VAD]    RTFx: 126.9x faster than real-time
-[23:26:10.167] [INFO] [VAD]    Files Processed: 24
-[23:26:10.167] [INFO] [VAD]    Avg Time per File: 0.115s
+Timing Statistics:
+[18:56:31.208] [INFO] [VAD]    Total processing time: 0.29s
+[18:56:31.208] [INFO] [VAD]    Total audio duration: 351.05s
+[18:56:31.208] [INFO] [VAD]    RTFx: 1230.6x faster than real-time
+[18:56:31.208] [INFO] [VAD]    Audio loading time: 0.00s (0.6%)
+[18:56:31.208] [INFO] [VAD]    VAD inference time: 0.28s (98.7%)
+[18:56:31.208] [INFO] [VAD]    Average per file: 0.011s
+[18:56:31.208] [INFO] [VAD]    Min per file: 0.001s
+[18:56:31.208] [INFO] [VAD]    Max per file: 0.020s
+[18:56:31.208] [INFO] [VAD]
+VAD Benchmark Results:
+[18:56:31.208] [INFO] [VAD]    Accuracy: 96.0%
```

Comment:
just lowered threhsoold, could probably bump it up if we really want d 100%. threshold at 0.9 = 100%

Reply:
just pulled the mlpackage from HF and quantized that

Reply:
<img width="859" height="822" alt="bweng-Google Chrome-2025-09-14-at-10 46 01" src="https://github.com/user-attachments/assets/27c3fdb9-8d66-4ac4-8106-4ae70087be49" />